flink-formats/flink-compress/src/test/java/org/apache/flink/formats/compress/CompressionFactoryITCase.java

```diff
@@ -43,27 +43,24 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * Integration test case for writing bulk encoded files with the {@link StreamingFileSink} and
  * Hadoop Compression Codecs.
  */
-public class CompressionFactoryITCase extends AbstractTestBase {
+class CompressionFactoryITCase extends AbstractTestBase {
```

Comment:
This seems fishy given that it extends AbstractTestBase which is still on junit4.

Reply:
Hey, thanks -- I'll rebase on 1.16 and make this change to an `@ExtendsWith(MiniClusterResource)`